contracts/payment/PaymentSplitter.sol

```diff
@@ -34,50 +38,52 @@ contract PaymentSplitter {
     }
 
     /**
-     * @dev Payable fallback.
+     * @dev The ether received will be logged with `PaymentReceived` events. Note that these events are not fully
+     * reliable: it's possible for a contract to receive ether without triggering this function. This only affects the
```

Comment:
I wished to have some place to link here that explains this more in depth. What could that be?

Reply:
There's a warning in the Solidity documentation: https://solidity.readthedocs.io/en/develop/contracts.html#fallback-function (the last one).

Reply:
I'd quote that in the source as well as linking to it.

Reply:
Would you quote it in every place in OpenZeppelin that has this same problem?

Reply:
Hmm, the warning in the Solidity documentation is not as thorough as what I wanted.

Reply:
What other parts of OZ have this issue?

Reply:
Hm, none it seems.